packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -211,6 +207,8 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     };
   }
 
+  private _getContext = (): IScrollablePaneContext => this.context;
```

Comment:
Why is a separate getter needed here?

Reply:
The types for react 16.8 don't give type safety on context. This getter seemed like the cleanest way to get the type annotation propagated.

Reply:
Well that's annoying :(